src/libs/TransactionUtils.ts

```diff
@@ -200,7 +200,8 @@ function getTransaction(transactionID: string): Transaction | Record<string, nev
  * The comment does not have its modifiedComment counterpart.
  */
 function getDescription(transaction: Transaction): string {
-    return transaction?.comment?.comment ?? '';
+    // Casting this to string to avoid wrong data types being returned from the API, is shared in the code.
```

Comment:
What does the `is shared in the code` part of the comment mean? I would think something like this is good: 

```suggestion
    // Casting the description to string to avoid wrong data types (e.g. number) being returned from the API
```

Reply:
You are right it doesn't make much sense. I updated the PR based on your suggestion.